app/models/user.rb

```diff
@@ -37,7 +40,7 @@ class User < ApplicationRecord
   has_many :user_plan_changes
 
   def self.find_or_create_from_sso(payload)
-    email = payload.fetch("email")
+    email = payload.fetch("email")&.downcase
```

Comment:
Out of curiosity... I guess this is not a typo... what means `&`

Reply:
is the safe navigation operator, same as `?.` in javascript